demo/components/victory-boxplot-demo.js

```diff
@@ -76,6 +76,7 @@ export default class App extends React.Component {
         <VictoryChart style={chartStyle} domain={{ x: [0, 20], y: [0, 3] }}>
           <VictoryBoxPlot
             minLabels maxLabels
+            q1Labels={() => ""}
```

Comment:
did you mean to commit this?

Reply:
Yep!

Reply:
This is a necessary change if you want labels to show up when you click on them. They need to be defined ahead of time even if just with a throwaway function.